src/test/java/com/gojek/beast/protomapping/ProtoUpdateListenerTest.java

```diff
@@ -57,7 +58,8 @@ public void setUp() {
         System.setProperty("ENABLE_AUTO_SCHEMA_UPDATE", "false");
         stencilConfig = ConfigFactory.create(StencilConfig.class, System.getProperties());
         protoMappingConfig = ConfigFactory.create(ProtoMappingConfig.class, System.getProperties());
-        protoUpdateListener = new ProtoUpdateListener(protoMappingConfig, stencilConfig, stencilClient, protoMappingConverter, protoMappingParser, bqInstance, protoFieldFactory);
+        appConfig = ConfigFactory.create(AppConfig.class, System.getProperties());
+        protoUpdateListener = new ProtoUpdateListener(new ConfigStore(appConfig, stencilConfig, protoMappingConfig, null), stencilClient, protoMappingConverter, protoMappingParser, bqInstance, protoFieldFactory);
```

Comment:
you can pass appConfig as null if its not being used.

Reply:
Is appConfig required for using `ProtoParserWithRefresh`? if not lets remove this